### PR TITLE
Formatter: Allow {X} to work as format string, not only {:X}

### PR DESCRIPTION
**Pull request by mathias-lang-sociomantic**
A change on a rather obscure behavior.

### CODE REVIEW

File: src/ocean/text/convert/Formatter_test.d [hunk 1]
```diff
@@ -543,3 +543,10 @@ unittest
     const bool NO  = false;
     test(format("{} -- {}", YES, NO) == "true -- false");
 }
+
+// Test for a previous parser bug
```
Comment (gavin-norman-sociomantic) [inline src/ocean/text/convert/Formatter_test.d]: Was this regarded as a bug then?

Comment (mathias-lang-sociomantic) [reply src/ocean/text/convert/Formatter_test.d]: Not exactly. More like an unexpected and baseless limitation. I'll update the comment.

Comment (mathias-lang-sociomantic) [reply src/ocean/text/convert/Formatter_test.d]: Done

**Review by gavin-norman-sociomantic — APPROVED** [review]
Maybe release notes? It's basically a new feature.

**Comment by gavin-norman-sociomantic** [comment]
One more thought: is there documentation anywhere describing all of the format specifiers? If there is, it should be updated too.

**Comment by mathias-lang-sociomantic** [comment]
https://github.com/sociomantic-tsunami/ocean/issues/43

**Comment by mathias-lang-sociomantic** [comment]
Added release notes

**Comment by gavin-norman-sociomantic** [comment]
>#43

Bumped into current milestone!

File: relnotes/parser-extended.feature.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+## Simplified format specifier
+
+`ocean.text.convert.Formatter`
+
+The Formatter used to refuse format specifiers lacking a column, so `{:X}` had to be used instead of `{X}`.
```
Comment (gavin-norman-sociomantic) [inline relnotes/parser-extended.feature.md]: "column" -> "colon"?

Comment (mathias-lang-sociomantic) [reply relnotes/parser-extended.feature.md]: Fixed

**Review by gavin-norman-sociomantic — APPROVED** [review]

**Pull request merged**